sensing/autoware_pointcloud_preprocessor/schema/dual_return_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Dual Return Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "dual_return_outlier_filter": {
+      "type": "object",
+      "properties": {
+        "x_max": {
+          "type": "number",
+          "description": "Maximum of x in meters for `Fixed_xyz_ROI` mode",
+          "default": "18.0"
+        },
+        "x_min": {
+          "type": "number",
+          "description": "Minimum of x in meters for `Fixed_xyz_ROI` mode",
+          "default": "-12.0"
+        },
+        "y_max": {
+          "type": "number",
+          "description": "Maximum of y in meters for `Fixed_xyz_ROI` mode",
+          "default": "2.0"
+        },
+        "y_min": {
+          "type": "number",
+          "description": "Minimum of y in meters for `Fixed_xyz_ROI` mode",
+          "default": "-2.0"
+        },
+        "z_max": {
+          "type": "number",
+          "description": "Maximum of z in meters for `Fixed_xyz_ROI` mode",
+          "default": "10.0"
+        },
+        "z_min": {
+          "type": "number",
+          "description": "Minimum of z in meters for `Fixed_xyz_ROI` mode",
+          "default": "0.0"
+        },
+        "min_azimuth_deg": {
+          "type": "number",
+          "description": "The left limit of azimuth in degrees for `Fixed_azimuth_ROI` mode",
+          "default": "135.0",
+          "minimum": 0,
+          "maximum": 360
+        },
+        "max_azimuth_deg": {
+          "type": "number",
+          "description": "The right limit of azimuth in degrees for `Fixed_azimuth_ROI` mode",
+          "default": "225.0",
+          "minimum": 0,
+          "maximum": 360
+        },
+        "max_distance": {
+          "type": "number",
+          "description": "The limit distance for in meters `Fixed_azimuth_ROI` mode",
```

Comment:
???

Reply:
thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8475/commits/0d6b0bf74ad5ac6e83d5461c7e6d7479fe06e11f